js/src/components/google-ads-account-card/disconnect-account.js

```diff
@@ -9,10 +9,26 @@ import { useState, useCallback } from '@wordpress/element';
  */
 import { useAppDispatch } from '.~/data';
 import AppButton from '.~/components/app-button';
+import useEventPropertiesFilter from '.~/hooks/useEventPropertiesFilter';
+import { FILTER_ONBOARDING } from '.~/utils/tracks';
 
+/**
+ * Clicking on the button to disconnect the Google Ads account.
+ *
+ * @event gla_ads_account_disconnect_button_click
+ * @property {string} [context] Indicates the place where the button is located.
+ * @property {string} [step] Indicates the step in the onboarding process.
```

Comment:
Same as above, do you think it'd be better to add more props in the doc?

Reply:
See https://github.com/woocommerce/google-listings-and-ads/pull/2421#discussion_r1624242157